### ISSUE TITLE
Optimal Hyperparameters

### ISSUE THREAD

**Issue by SaiKeshav** [siqims/ARC-Challenge#1]
Thanks a lot for sharing your code! If it is not too much to ask, could you also share the optimal hyperparameters for the model (learning_rate, batch_size, and the number of epochs) for the BERT base and large model? It would be really helpful for me - to replicate your results on the ARC datasets.

Thanks in advance!
Keshav

**Comment by intersun** [comment]
learning_rate, batch_size, and the number of epochs is set as  (1e-5, 12, 4), the optimal model is selected by the accuracy on dev set.  The selected model is always from epoch 2 or 3 though.
